gradle/libs.versions.toml

```diff
@@ -1,47 +1,47 @@
 [versions]
 javaVersion = "17"
 
-kotlinVersion = "2.0.21" # Kotlin and Dokka versions should have the same minor version to support the same kotlin version
+kotlinVersion = "2.1.21" # Kotlin and Dokka versions should have the same minor version to support the same kotlin version
 dokkaVersion = "2.0.0" # Kotlin and Dokka versions should have the same minor version to support the same kotlin version
-agpVersion = "8.10.0"
-ktlintVersion = "12.2.0"
+agpVersion = "8.13.0"
+ktlintVersion = "13.1.0"
```

Comment:
Was this aligned with everyone?
I might have missed it, but I had impression that we use different AGP version: "8.12.1"